src/tables.ts

```diff
@@ -0,0 +1,1526 @@
+  accSagittal2TinasDown: { code: 'accSagittal2TinasDown', parenRightPaddingAdjustment: -1 },
+  accSagittal3TinasUp: { code: 'accSagittal3TinasUp', parenRightPaddingAdjustment: -1 },
+  accSagittal3TinasDown: { code: 'accSagittal3TinasDown', parenRightPaddingAdjustment: -1 },
+  accSagittal4TinasUp: { code: 'accSagittal4TinasUp', parenRightPaddingAdjustment: -1 },
+  accSagittal4TinasDown: { code: 'accSagittal4TinasDown', parenRightPaddingAdjustment: -1 },
+  accSagittal5TinasUp: { code: 'accSagittal5TinasUp', parenRightPaddingAdjustment: -1 },
+  accSagittal5TinasDown: { code: 'accSagittal5TinasDown', parenRightPaddingAdjustment: -1 },
+  accSagittal6TinasUp: { code: 'accSagittal6TinasUp', parenRightPaddingAdjustment: -1 },
+  accSagittal6TinasDown: { code: 'accSagittal6TinasDown', parenRightPaddingAdjustment: -1 },
+  accSagittal7TinasUp: { code: 'accSagittal7TinasUp', parenRightPaddingAdjustment: -1 },
+  accSagittal7TinasDown: { code: 'accSagittal7TinasDown', parenRightPaddingAdjustment: -1 },
+  accSagittal8TinasUp: { code: 'accSagittal8TinasUp', parenRightPaddingAdjustment: -1 },
+  accSagittal8TinasDown: { code: 'accSagittal8TinasDown', parenRightPaddingAdjustment: -1 },
+  accSagittal9TinasUp: { code: 'accSagittal9TinasUp', parenRightPaddingAdjustment: -1 },
+  accSagittal9TinasDown: { code: 'accSagittal9TinasDown', parenRightPaddingAdjustment: -1 },
+  accSagittalFractionalTinaUp: { code: 'accSagittalFractionalTinaUp', parenRightPaddingAdjustment: -1 },
+  accSagittalFractionalTinaDown: { code: 'accSagittalFractionalTinaDown', parenRightPaddingAdjustment: -1 },
+  accidentalNarrowReversedFlat: { code: 'accidentalNarrowReversedFlat', parenRightPaddingAdjustment: -1 },
+  accidentalNarrowReversedFlatAndFlat: {
+    code: 'accidentalNarrowReversedFlatAndFlat',
+    parenRightPaddingAdjustment: -1,
+  },
+  accidentalWilsonPlus: { code: 'accidentalWilsonPlus', parenRightPaddingAdjustment: -1 },
+  accidentalWilsonMinus: { code: 'accidentalWilsonMinus', parenRightPaddingAdjustment: -1 },
+};
+
+export const Flow = {
+  STEM_WIDTH: 1.5,
+  STEM_HEIGHT: 35,
+  STAVE_LINE_THICKNESS: 1,
+  RESOLUTION: RESOLUTION,
+
+  /**
+   * Customize this to choose a different music font.
+   * For example: Vex.Flow.DEFAULT_FONT_STACK = [Fonts.Petaluma, Fonts.Custom];
+   */
+  DEFAULT_FONT_STACK: [Fonts.Bravura, Fonts.Gonville, Fonts.Custom],
+  DEFAULT_NOTATION_FONT_SCALE: 39,
+  DEFAULT_TABLATURE_FONT_SCALE: 39,
+  SLASH_NOTEHEAD_WIDTH: 15,
+  STAVE_LINE_DISTANCE: 10,
+
+  // HACK:
+  // Since text origins are positioned at the baseline, we must
+  // compensate for the ascender of the text. Of course, 1 staff space is
+  // a very poor approximation.
+  //
+  // This will be deprecated in the future. This is a temporary solution until
+  // we have more robust text metrics.
+  TEXT_HEIGHT_OFFSET_HACK: 1,
+
+  /* Kerning (DEPRECATED) */
+  IsKerned: true,
+  Element: Element,
+  Fraction: Fraction,
+  Renderer: Renderer,
+  Formatter: Formatter,
```

Comment:
Can we update these imported elements to `index.ts`?

Reply:
Typescript does not like to include additional properties to the variable once it is declared.

Reply:
Putting it in here makes it really messy. You can do this:

```typescript
// tables.ts
export const Flow: Record<any, any> = {
  DEFAULT_FONT_STACK: ...,
 // ...
};
```

```typescript
// index.ts
import {Flow} from './tables'

Flow.Element = Element;
```

Here's another option: in this file call it `Tables` instead of `Flow`.
```typescript
// tables.ts
export const Tables = {
  DEFAULT_FONT_STACK: ...,
 // ...
};
```

Then inside `index.ts` you can declare `Flow` like so:

```typescript
// index.ts
import {Tables} from './tables'
import {Element} from './element'
// ...

export const Flow = {
  Element: Element,
  // ...
  ...Tables
}
```

I like this second approach better, if it works.

Reply:
Me too, I will give it a try. All the other PRs are also ready for review :)

Reply:
There is a problem, `Flow` is also required by `Vex`. I could do an additional file called `flow.ts` following the approach above:
`table.ts` exports `Table`
`flow.ts` imports `table.ts` and exports `Flow`
`vex.ts` imports `flow.ts` and exports `Vex`

I could remove `index.ts` and declare `vex.ts` the entry point of the library as well.

Reply:
Thanks Rodrigo. Additional `flow.ts` sounds good. Keep entry point as `index.ts` please.